src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenPaymentMade.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.divorce.citizen.event;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.ccd.sdk.api.callback.AboutToStartOrSubmitResponse;
+import uk.gov.hmcts.divorce.common.service.SubmissionService;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+import uk.gov.hmcts.divorce.payment.model.PaymentStatus;
+
+import java.util.Collections;
+import java.util.List;
+
+import static uk.gov.hmcts.ccd.sdk.type.YesOrNo.NO;
+import static uk.gov.hmcts.ccd.sdk.type.YesOrNo.YES;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingDocuments;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingPayment;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CASE_WORKER;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CITIZEN;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SUPER_USER;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+import static uk.gov.hmcts.divorce.divorcecase.validation.ApplicationValidation.validateSubmission;
+import static uk.gov.hmcts.divorce.payment.model.PaymentStatus.IN_PROGRESS;
+import static uk.gov.hmcts.divorce.payment.model.PaymentStatus.SUCCESS;
+
+@Component
+@Slf4j
+public class CitizenPaymentMade implements CCDConfig<CaseData, State, UserRole> {
```

Comment:
maybe would've been better if you renamed the other class CitizenPaymentMade so that the Git diff is more meaningful

Reply:
i tried that and it didnt work